tests/queries/0_stateless/02674_space_function.reference

```diff
@@ -0,0 +1,25 @@
+   
+   3
+  2
+ 1
+0
+
+            12
+          10
+    4
+     5
+
+    4
+                     21
+         9
+       7
+
+                                                       56
+                    20
+     5
+       7
+
+NULL
+NULL
+NULL
+NULL                                                       
```

Comment:
Do you generate reference manully? Check https://clickhouse.com/docs/en/development/tests/.

Reply:
yes,  I will look at this. And thanks for your reviews.

Reply:
You can generate reference by `clickhouse-client -nm < test.sql > test.reference`